### DISCUSSION TITLE
Add [inv]

### DISCUSSION THREAD

**Discussion by SkyDude0** [Zrips/CMI#6934] [Enhancement]
Like [i] , add a way to see inventory of player in chat with like [inv]

**Comment by mrfloris** [comment]
This plugin has support to do that, https://www.spigotmc.org/resources/interactivechat-show-items-inventory-in-chat-custom-chat-keywords-bungee-velocity-support.75870/, if you use their discord addon. It then lets you put inventories etc as images in your chat. 

Would be cool indeed if cmi has [inv] in the chat.

**Comment by TheMartyCZ** [comment]
I would like to support this proposal. Since CMI can show the item in the chat, it wouldn't be bad if it could also show the inventory.
